jwql/instrument_monitors/common_monitors/dark_monitor.py

```diff
@@ -649,23 +649,14 @@ def overplot_bad_pix(self, pix_type, coords, values):
             values = []
 
         sources[pix_type] = ColumnDataSource(data=dict(pixels_x=coords[0],
-                                                       pixels_y=coords[1],
-                                                       values=values
```

Comment:
Removing the values variable here is okay by me if the list is empty. Do we use this variable else where in the module?

Reply:
This is just for the ColumnDataSource. The pixel values are not included in the plot. So I'm removing them here just because it is an unused variable.

Reply:
Thanks!